frontend/src/static/css/styles.scss

```
@@ -1690,7 +1690,7 @@ video.show-video {
 }
 
 .film-title {
-	font-family: 'Facultad', sans-serif;
+	font-family: 'Barlow Semi Condensed', 'Arial Narrow', 'Arial', sans-serif;
```

Stylelint (17.11.1)

[error] Expected no quotes around "Arial" (font-family-name-quotes)

(font-family-name-quotes)